types/benchmark_test.go

```diff
@@ -0,0 +1,21 @@
+package types
+
+import "testing"
+
+var r uint64 = 0xdeadbeef
+
+// Pseudo-random number generator adapted from
+// https://github.com/dgryski/trifles/blob/master/fastrand/fastrand.go
+func rand() int64 {
```

Comment:
You may just import the package: https://github.com/coocood/rtutil/blob/master/rtutil.go#L32

Reply:
I tried it, seems working.